src/example.js

```diff
@@ -1,4 +1,6 @@
-(function() {
+(function(bla) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "bla". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)